theano/gpuarray/elemwise.py

```diff
@@ -74,11 +74,12 @@ def make_node(self, *inputs):
         except MethodNotDefined:
             pass
         try:
-            support_code = self.scalar_op.c_support_code()
-            #if (support_code.strip() != "#define THEANO_MACRO_MOD(x,y) (x % y)" and
+            pass
+            # support_code = self.scalar_op.c_support_code()
+            # if (support_code.strip() != "#define THEANO_MACRO_MOD(x,y) (x % y)" and
             #       support_code.strip() != ""):
-                # The macro is fine, the C++ struct is not.
-                #raise SupportCodeError(support_code)
+            #    The macro is fine, the C++ struct is not.
+            #    raise SupportCodeError(support_code)
```

Comment:
Can you rebase the PR and take the current code in master for this part? We have a better fix for this in the master. Also, currently, this PR can't be merged, the rebase should fix that.